GetPairNumberFromColors.js

```diff
@@ -0,0 +1,31 @@
+const MajorColors = [

```

Comment:
Avoid duplicating the list of colors in two files. We'll always need to make additions corrections in two places, which may not be evident to other developers.